app/models/topic.rb

```diff
@@ -0,0 +1,18 @@
+# == Schema Information
+#
+# Table name: topics
+#
+#  id         :integer          not null, primary key
+#  name       :string
```

Comment:
Are you going to add validations in a next PR? I think that the name should be mandatory